source/app/models/game.rb

```diff
@@ -0,0 +1,3 @@
+class Game < ActiveRecord::Base
+
```

Comment:
You should declare your relations here. It would let you use `winner` rather than `winner_id` in the rest of your code. As is, you're duplicating a lot of the work that ActiveRecord does for you.